wire-ios-mono.xcworkspace/xcshareddata/xcdebugger/Breakpoints_v2.xcbkptlist

```diff
@@ -16,15 +16,6 @@
             symbolName = "-[UIViewController presentViewController:animated:completion:]"
             moduleName = "">
             <Locations>
-               <Location
-                  uuid = "19AA3E78-E6B0-4B09-B06D-EF3DF0CDEF5C - e3f40a65cc9014d3"
-                  shouldBeEnabled = "Yes"
-                  ignoreCount = "0"
-                  continueAfterRunningActions = "No"
-                  symbolName = "-[UIViewController presentViewController:animated:completion:]"
-                  moduleName = "UIKitCore"
-                  usesParentBreakpointCondition = "Yes">
-               </Location>
```

Comment:
@caldrian I saw this on my branches too sometimes. Think it's save to remove, what is your opinion?